### PR TITLE
feat(wkt): WKB writer

### PR DESCRIPTION
**Pull request by ibgreen**

### CODE REVIEW

File: modules/wkt/src/lib/encode-wkb.ts [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+    default:
+      throw new Error('stringify requires a valid GeoJSON Feature or geometry object as input');
+  }
+}
+
+/** Calculate the binary size (in the WKB encoding) of a specific geojson geometry */
+function getGeometrySize(geometry: Geometry): number {
+  switch (geometry.type) {
+    case 'Point':
+      return getPointSize(geometry as Point);
```
Comment (kylebarron) [inline modules/wkt/src/lib/encode-wkb.ts:61]: Do you need these casts? I would've expected TS to understand this discriminated union in the `case`

File: modules/wkt/test/wkb/encode-wkb.spec.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
+import test from 'tape-promise/tape';
```
Comment (ibgreen) [inline modules/wkt/test/wkb/encode-wkb.spec.ts:1]: Do we need any attribution?

File: modules/wkt/test/wkb/encode-wkb.spec.ts [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+const WKB_2D_TEST_CASES = '@loaders.gl/wkt/test/data/wkb-testdata2d.json';
+const WKB_2D_NAN_TEST_CASES = '@loaders.gl/wkt/test/data/wkb-testdata2d-nan.json';
+const WKB_Z_TEST_CASES = '@loaders.gl/wkt/test/data/wkb-testdataZ.json';
+const WKB_Z_NAN_TEST_CASES = '@loaders.gl/wkt/test/data/wkb-testdataZ-nan.json';
+
+interface TestCase {
+  geometry: string;
+  wkt: string;
+  wkb: string;
+  ewkb: string;
```
Comment (ibgreen) [inline modules/wkt/test/wkb/encode-wkb.spec.ts:16]: Do we use all of this? Comments to help navigate the alphabet soup?

**Review by ibgreen — COMMENTED** [review]
We probably want a doc and a mention in what's new.

File: modules/wkt/test/wkb/encode-wkb.spec.ts (hunk 2 above)
Comment (kylebarron) [reply modules/wkt/test/wkb/encode-wkb.spec.ts:1]: The test data is the same as what's used for for `WKBLoader`, an attribution sentence is here: https://github.com/visgl/loaders.gl/blob/master/modules/wkt/test/data/README.md

File: modules/wkt/test/wkb/encode-wkb.spec.ts (hunk 3 above)
Comment (kylebarron) [reply modules/wkt/test/wkb/encode-wkb.spec.ts:16]: I was planning to put this in a `utils.ts` to share with `parse-wkb.spec.ts`

**Pull request merged**